features/manage-position/MethodSelector.tsx

```diff
@@ -59,19 +59,19 @@ const MethodSelector = ({ method, handleChange }: IProps) => {
           <MenuItem value={"deposit"}>
             <ListItemText
               primary="Deposit"
-              secondary="Deposit additional collateral."
+              secondary="Top up your position's collateral."
```

Comment:
Is "Top up your position's" preferable to "Deposit additional"? I find the former more precise language and less casual. Depends what vibe we're going for I guess

Reply:
Oh well, I guess this is `secondary` text. I recommend changing "Top up" with "Add to" to parallel the "Remove" language. Also remove "your".

Recommendation: "Add to position collateral"

Reply:
ye, I changed this because the menu was so repetitive. All the actions said basically the same thing as the primary:
![image](https://user-images.githubusercontent.com/12886084/88572113-efec1e00-d03e-11ea-8362-dbcbaf173a83.png)

Will change to your recommendation.